### PR TITLE
Update dateutil req version to 3.0.0.

### PR DESCRIPTION
**Pull request by smarlowucf**
Add all requires as buildrequires in spec.

### CODE REVIEW

File: package/mash.spec [hunk 1]
```diff
@@ -27,6 +27,27 @@ Source:         mash-%{version}.tar.gz
...
+BuildRequires:  python3-azure-mgmt-storage
+BuildRequires:  python3-azure-storage
+BuildRequires:  python3-boto3
+BuildRequires:  python3-cryptography
+BuildRequires:  python3-jsonschema
+BuildRequires:  python3-PyYAML
+BuildRequires:  python3-PyJWT
+BuildRequires:  python3-amqpstorm >= 2.4.0
+BuildRequires:  python3-APScheduler >= 3.3.1
+BuildRequires:  python2-tzlocal
```
Comment (rjschwei) [inline package/mash.spec]: Typo, python2 ?

Comment (smarlowucf) [reply package/mash.spec]: Not a typo. tzlocal is required in dep tree and for some reason python3-tzlocal is packaged as a symlink to python2-tzlocal with no explicit requirement. So this is needed for python3-tzlocal to work.

Comment (smarlowucf) [reply package/mash.spec]: Will look at fixing package in dev

**Comment by smarlowucf** [comment]
Remove python2 requirement. Package fixed.

**Pull request merged**